typescript-client-sdk/client.ts

```diff
@@ -40,10 +40,11 @@ export class HathoraClient {
     return res.data.token;
   }
 
```

Comment:
while you're in this file, add a method called: `generateSiweNonce(): Promise<{ nonce: string, nonceToken: string }>` that makes a request to the auth-server `/nonce` endpoint.